src/index.html

```diff
@@ -4,9 +4,333 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="index.html">
+        <img
+          src="./images/logo.png"
+          alt="Moyo-logo"
+          class="logo"
+        >
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <a href="index.html" class="nav__link is-active">
+            <li>Apple</li>
+          </a>
```

Comment:
```suggestion
          <li class="nav__item">
            <a href="#apple" class="nav__link is-active">
              Apple
            </li>
          </li>
```
- Should be vise versa. Inside the `ul` element you need to place `li` elements and inside `li` you can place other
- And don't use `index.html` as a `href` attribute, in this case, the page reloading when the user clicks

Fix in all places